src/librustdoc/html/static/main.js

```diff
@@ -488,10 +488,10 @@ function hideThemeButtonState() {
         hideThemeButtonState();
     }
 
-    var disableShortcuts = getSettingValue("disable-shortcuts") === "true";
+    var disableShortcuts = getSettingValue("disable-shortcuts") == "true";
```

Comment:
Actually, this comparison must keep using `===` because when you store something in the local storage, it is always converted to string.